tutorials/scripting/singletons_autoload.rst

```diff
@@ -94,6 +94,27 @@ be accessed directly in GDScript, without requiring ``get_node()``:
 
    PlayerVariables.health -= 10
 
+The **Enable** column has no effect in C# code. However, if the singleton is a
+C# script, a similar effect can be achieved by adding a static property:
```

Comment:
I think it's awkward enough having a GDScript-only code tab switcher, so I don't think it's great to also add a C#-only one. If there's some way to put it all under C#/GDScript tabs, it seems like that would be nicer? I don't imagine it would be ok to change all the explanation into a code comment, though, and that's the only way I know how to do it.

Another idea is to put this example in the C# docs somewhere (in the "C# language features" page?) and add a note here that links to it. I don't really know how important it is to document C# stuff up front vs. siloing it.

I don't have any strong preferences here, just want to get this in the docs somewhere. 😄

Reply:
Code can't be translated so I don't think we should move the explanation to code comments.

As for tab switchers with only one tab, we can add code snippets without tabs. See: https://github.com/godotengine/godot-docs/blob/67b17958ca317cec3230d09ed01f45e799ecf949/tutorials/scripting/c_sharp/c_sharp_basics.rst?plain=1#L176

But this is from the C# basics page so one could assume that all the code snippets are in C#, therefore there's no need to indicate the language. In the singletons page we have snippets in GDScript too, so maybe the tab is useful as a language indicator idk.

Reply:
Yeah, language indicator is how I interpreted the single tag, and as long as we can't make whole sections swap, IMO it does its job fine. Interesting note about translation, didn't realize that but it makes sense.